src/App.vue

```diff
@@ -91,10 +96,18 @@ export default class App extends Vue {
 
   private filingData: Array<FilingDataIF> = []
   private totalFee: number = 0
+  private entityType: string = 'CP'
+
+  @Action('setEntityType') setEntityType!: ActionBindingIF
+  @Action('setCurrentDate') setCurrentDate!: ActionBindingIF
+  @Action('setCertifyStatementResource') setCertifyStatementResource!: ActionBindingIF
 
   // Lifecycle event
   private created (): void {
     this.filingData.push({ filingTypeCode: FilingCodes.INCORPORATION_BC, entityType: EntityTypes.BCOMP })
+    this.setEntityType(this.entityType)
```

Comment:
Will this be moved to step1?

If so, it might make sense to have a watcher for entityType and then set the CertifyStatementResource in the watcher.

Thoughts?

Reply:
That will move from App.vue but as part of another pr which will have the finalized text for bcomp